config/Config-images.in

```diff
@@ -17,6 +17,7 @@ menu "Target Images"
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_lantiq
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_mpc85xx
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ramips
+			default TARGET_INITRAMFS_COMPRESSION_XZ if TARGET_ath79_generic_DEVICE_avm_fritzx490-wasp
```

Comment:
The build bot will probably build this with `CONFIG_TARGET_INITRAMFS_COMPRESSION_NONE=y` 
Will such images work or are they too big?

Reply:
I think you should remove this line